spec/views/groups/new.html.haml_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe "groups/new", type: :view do
+  before(:each) do
+    assign(:group, Group.new(
+      :category => "MyString",
```

Comment:
Indent the first parameter one step more than `Group.new(`.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.